content/en/about/supported-releases/index.md

```diff
@@ -5,19 +5,66 @@ weight: 35
+|Patches           | Same as the corresponding Snapshot/LTS release           | Users are encouraged to adopt patch releases as soon as they are available for a given release.
+
+You can find available releases on the [releases page](https://github.com/istio/istio/releases),
+and if you're the adventurous type, you can learn about our development builds on the [development builds wiki](https://github.com/istio/istio/wiki/Dev%20Builds).
+You can find high-level releases notes for each LTS release [here](/news).
+
+## Naming scheme
+
+Our naming scheme for LTS releases is:
+
+{{< text plain >}}
+<major>.<minor>.<LTS patch level>
+{{< /text >}}
+
+where `<minor>` is increased for every LTS release, and `<LTS patch level>` counts the number of patches for the
+current LTS release. A patch is usually a small change relative to the LTS.
+
+For snapshot releases, our naming scheme is:
+
+{{< text plain >}}
+<major>.<minor>-alpha.<sha>
+{{< /text >}}
+
+where `<major>.<minor>` represent the next LTS, and
+`<sha>` represents the git commit the release is built from.
+
 
 ## Support status of Istio releases
 
-| Version         | Currently Supported   | Release Date      | End of Life       | Supported Kubernetes Versions |
-|-----------------|-----------------------|-------------------|-------------------|-------------------------------|
-| master          | No, development only  |                   |                   |                               |
-| 1.8             | Yes                   | November 10, 2020 |                   | 1.16, 1.17, 1.18, 1.19        |
-| 1.7             | Yes                   | August 21, 2020   |                   | 1.16, 1.17, 1.18              |
-| 1.6             | No                    | May 21, 2020      | November 23, 2020 | 1.15, 1.16, 1.17, 1.18        |
-| 1.5 and earlier | No                    |                   |                   |                               |
+| Version         | Currently Supported   | Release Date      | End of Life       | Supported Kubernetes Versions | Untested, but may work on Kubernetes Versions |
+|-----------------|-----------------------|-------------------|-------------------|-------------------------------|-----------------------------------------------|
+| master          | No, development only  |                   |                   |                               |                                               |
+| 1.9             | N/A                   | TBD               |                   | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
+| 1.8             | Yes                   | November 10, 2020 |                   | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
+| 1.7             | Yes                   | August 21, 2020   |                   | 1.16, 1.17, 1.18              | 1.15                                          |
```

Comment:
We know the EOL should be 3 months post 11/10/20 although we aren't there yet.

Reply:
I recommend putting the expected EOL date for the ones that are active (with a comment that its expected)

Reply:
Added, let me know what you think.